package.json

```
@@ -3,6 +3,9 @@
   "version": "0.0.0",
   "private": true,
   "license": "Apache-2.0",
+  "engines": {
+    "node": "16"
```

I think we want one of these instead:

```suggestion
    "node": ">= 16.0.0"
```
```suggestion
    "node": ">= 16.20.2"
```

---

It can't hurt to have folks running 18 or 20 if we specify that we use 16 on CI, but I think it would be good for us to keep it to the same major version in engines if we're gonna be strict about it 🤔

---

Doesn't `"node": "16"` imply only `16.0.0` though? I guess the syntax I was looking for was `"node": ">=16.0.0 <17"` ([example](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines))